examples/skip_plot_map_in_3d.py

```diff
@@ -40,4 +38,11 @@
                 frame='heliocentricinertial')
 plotter.plot_line(line)
 
-plotter.show(cpos=(-100, 0, 0))
+# Define a SkyCoord for to set the positon of the camera
+camera_position = SkyCoord(0*u.deg, 0*u.deg, 8*const.R_sun, obstime=m.observer_coordinate.obstime, frame=frames.HeliographicStonyhurst)
```

Comment:
Could we change this to `m.observer_coordinate` instead? It's simpler, and I think a common use case.

Reply:
![image](https://user-images.githubusercontent.com/50923538/120929849-51255900-c708-11eb-8e85-7a57a0c4361e.png)
That produces a plot really far away, is this the right behaviour?

Reply:
Yes, this looks right, since the default view angle (the width of the camera) is 30 degrees - https://docs.pyvista.org/core/camera.html#pyvista.Camera.view_angle. We should add a `view_angle` property to our `Plotter` that accepts a `u.deg` quantity as input, and sets the `view_angle` property on the pyvista plotter. Then it can be set to a smaller value to "zoom in" to the image. But we can do all this in the next PR.

Reply:
Ah, okay. I'll adjust that later then